pebblo/app/service/classification.py

```diff
@@ -106,6 +101,15 @@ def process_request(self):
                 return PebbloJsonResponse.build(
                     body={"error": "Input data is missing"}, status_code=400
                 )
+            if not req.mode:
+                req.mode = config_details.get("classifier", {}).get(
```

Comment:
Given the `/api/v1/classify` is a classifier utility wrapper, let's not apply the pebblo-server configuration for this API.